easybuild/easyconfigs/p/PyTorch-Lightning/PyTorch-Lightning-1.8.2-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-Lightning'
+version = '1.8.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://pytorchlightning.ai'
+description = "PyTorch Lightning is the lightweight PyTorch wrapper for ML researchers."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('Python', '3.10.4'),
+    ('PyTorch', '1.12.0', versionsuffix),
+    ('tqdm', '4.64.0'),
+    ('tensorboard', '2.10.0'),
+    ('typing-extensions', '4.3.0'),
+]
+
+exts_list = [
+    ('fsspec', '2022.11.0', {
+        'checksums': ['259d5fd5c8e756ff2ea72f42e7613c32667dc2049a4ac3d84364a7ca034acb8b'],
+    }),
+    ('pyDeprecate', '0.3.2', {
+        'modulename': 'deprecate',
+        'checksums': ['d481116cc5d7f6c473e7c4be820efdd9b90a16b594b350276e9e66a6cb5bdd29'],
+    }),
+    ('torchmetrics', '0.10.3', {
+        'checksums': ['9e6ab66175f2dc13e246c37485b2c27c77931dfe47fc2b81c76217b8efdc1e57'],
+    }),
+    ('fire', '0.4.0', {
```

Comment:
```suggestion
    # stick to termcolor 1.x, to avoid hatchling required dependency
    ('termcolor', '1.1.0', {
        'checksums': ['1d6d69ce66211143803fbc56652b41d73b4a400a2891d7bf7a1cdf4c02de613b'],
    }),
    ('fire', '0.4.0', {
```